pkg/alertmanager/amcfg.go

```diff
@@ -97,6 +97,14 @@ type inhibitRule struct {
 type receiver struct {
 	Name             string             `yaml:"name" json:"name"`
 	PagerdutyConfigs []*pagerdutyConfig `yaml:"pagerduty_configs,omitempty" json:"pagerduty_configs,omitempty"`
+	WebhookConfigs   []*webhookConfig   `yaml:"webhook_configs,omitempty" json:"webhook_configs,omitempty"`
+}
+
+type webhookConfig struct {
+	VSendResolved bool              `yaml:"send_resolved" json:"send_resolved"`
```

Comment:
```suggestion
	SendResolved bool              `yaml:"send_resolved" json:"send_resolved"`
```

Reply:
@paulfantom should I change it in `pagerdutyConfig` as well? I was not sure why is it called in this way for pd, so just reused this variable name.

Reply:
Seems like `VSendResolved` is a pattern used in alertmanager: https://github.com/prometheus/alertmanager/blob/7d28e5b876df41844f796ea7a49cad1b6207aa7b/config/notifiers.go#L31. Let's leave it as is then.